pages/dbfactory/dbfactory.js

```diff
@@ -9,14 +9,16 @@ export class DBfactory extends Component {
   constructor (props) {
     super(props)
     this.state = { people: 1, ops: 1 }
-    this.queryHandler = this.queryHandler.bind(this)
+    this.handleQuery = this.handleQuery.bind(this)
   }
-  queryHandler = (event) => {
+
+  handleQuery = (event) => {
```

Comment:
Hi Andrew, handleQuery is a fat arrow function so no need to bind function anymore